db/models.py

```diff
@@ -1 +1,26 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
```

Comment:
You can add a constant variable with races and parameter choices in the "name" field. But it works anyway. It's up to you